### DISCUSSION TITLE
How to check the list of registered language of textarea?

### DISCUSSION THREAD

**Discussion by Tim-Cao** [Textualize/textual#3393] [Q&A] [answered]
I would like to check the list of registered language of textarea

```
        yield TextArea(TEXT, language="python")
```

**Comment by davep** [comment, marked as answer]
I would have thought that [`TextArea.available_languages`](https://textual.textualize.io/widgets/text_area/#textual.widgets._text_area.TextArea.available_languages) would be the thing to look to.

**Reply by Tim-Cao** [reply]
You're right. BUILTIN_LANGUAGES
